src/app.css

```diff
@@ -240,23 +240,15 @@ mark {
   color: #777;
 }
 
-#structured-result aside {
+#structured-result {
   display: flex;
   justify-content: center;
   align-items: center;
   flex-direction: column;
 }
 
-#structured-result img {
+#structured-result .image {
   margin: 1rem auto;
-  animation: fadeIn 150ms;
-}
-
-@keyframes fadeIn {
-  from {
-    opacity: 0;
-  }
-  to {
-    opacity: 1;
-  }
+  height: 128px;
```

Comment:
This distorts images though, so not sure if better. I think we can remove the two following lines.

Reply:
I only give it a height, the width is flexible no?